src/interfaces/decorator-metadata-bag.interface.ts

```diff
@@ -0,0 +1,27 @@
+import { TypeMetadata, ExposeMetadata, ExcludeMetadata, TransformMetadata } from '.';
+
+/**
+ * Information assigned to a class property or class (constructor) via decorators.
+ */
+export interface DecoratorMetaDataBag<T> {
+  /**
+   * Optional metadata assigned through the @Expose decorator.
+   */
+  expose: ExposeMetadata[];
+
+  /**
+   * Optional metadata assigned through the @Exclude decorator.
+   */
+  exclude: ExcludeMetadata[];
+  /**
```

Comment:
Missing blank line between property and doc string.

Reply:
Code is WIP.

Reply:
Ok